### DISCUSSION TITLE
v8.0.0-alpha.0

### DISCUSSION THREAD

**Discussion by nicolo-ribaudo** [babel/babel#15805] [Releases]
Super early pre-release of Babel 8 :)

It's currently known to not work with:
- `@babel/cli` - see https://github.com/babel/babel/issues/15803
- `babel-loader` - see https://github.com/babel/babel/issues/15804

It is known to work with:
- `@rollup/plugin-babel`

`peerDependencies` are all incorrect, so use the npm flag `--legacy-peer-deps` if you want to test it.

If you find any big problems please report them in this discussion and I'll take care of organizing them in proper issues.

<hr /><em>This discussion was created from the release <a href='https://github.com/babel/babel/releases/tag/v8.0.0-alpha.0'>v8.0.0-alpha.0</a>.</em>

**Comment by Nantris** [comment]
You guys should codename this version El Dorado. I can't believe it's happening. Great work!

**Comment by nyngwang** [comment]
I just realized that my #16241 was merged two days ago and that Babel 8 alpha was already released last year. Anyway, I'm happy that somehow participated in a tiny bit of the improvement of Babel. Waiting for the stable release 👍

**Reply by liuxingbaoyu** [reply]
https://github.com/babel/babel/releases/tag/v8.0.0-alpha.7 :).

**Reply by nyngwang** [reply]
@liuxingbaoyu Sorry for me being stupid. Did you mean that v8.0.0-alpha is actually what I just said a stable release? Then I will try to upgrade my side project to try it today!

**Reply by liuxingbaoyu** [reply]
No, it's not a stable release yet, but it already contains that PR. Welcome to try it!

**Reply by nicolo-ribaudo** [reply]
7.24.0 was also released stable yesterday, including that PR :)

**Reply by nyngwang** [reply]
@nicolo-ribaudo I really appreciate your implementation of the issue! I hope Babel can win the battle among those JS build tools with the incoming Babel 8 stable release!

**Reply by nyngwang** [reply]
@nicolo-ribaudo Is it possible that #16329 to land on Babel v7 recently?
